### PR TITLE
Jesters/Mhoire Armory and Associated Quests

### PR DESCRIPTION
**Pull request by sLackeR-hub**
**Needs 00EA (Mhoires Armory) added to nodrop / nolog list in corpse.cs**

**Created Jesters / Mhoire Armory and associated Quests**

**Added**
NPCs -
Jester (Both North/South)
Danira the Dusty
Zho Heishan
Lacarra Avanthi
Lady Aerfalle

Creatures-
Corrupted Maiden
Spectral Dread
Forge Golem
Grave Rat
Wight Captain
Wight Blade Sorcerer
Rock Wall
Armory Door
West/North/East/South Armory Captains
Blood Priestess
Imperial Guard / Archivist
K'nath(s)
Lost Menilesh Denizen
Menilesh Guard
Celestial Illuminator
Majordomo (North/South)
Tainted Desolation Moarsman
Lord Rytheran

Items -
Jester's Markers
Jester's Marbles 
Doors / Keys
Mhoire Armory Portal (Quest Restricted)
Garden's of Menilesh (Quest Restricted)
Rytheran's Journal (Translated/Untranslated)
Rytheran's Master Seal
Ancient Chest
K'nath Core(s)
Summoning Stones

Quests - 
Jester's Focuses
Lord Rytheran's Journal
Disturbance in the Ley Lines
Jester's Lost Marbles
Jester's Markers
Spectral Weapons

### CODE REVIEW

File: Database/Patches/2008-05-Introductions/8 QuestDefDB/JesterMarkerQuest.sql [hunk 1]
```diff
@@ -0,0 +1,64 @@
+DELETE FROM quest WHERE name = 'jestermarbleturnin';
+
```
Comment (ziang4891) [inline Database/Patches/2008-05-Introductions/8 QuestDefDB/JesterMarkerQuest.sql:2]: This file is duplicated in 2008-03-FoolishAmbition\8 QuestDefDB

File: Database/Patches/2008-05-Introductions/9 WeenieDefaults/Creature/Unsorted/37157 Door.sql [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (37157,   1, True ) /* Stuck */
+     , (37157,   6, False) /* AiUsesMana */
+     , (37157,  11, False) /* IgnoreCollisions */
+     , (37157,  12, True ) /* ReportCollisions */
+     , (37157,  13, False) /* Ethereal */
+     , (37157,  14, True ) /* GravityStatus */
+     , (37157,  19, True ) /* Attackable */
+     , (37157,  29, True ) /* NoCorpse */;
+
```
Comment (ziang4891) [inline Database/Patches/2008-05-Introductions/9 WeenieDefaults/Creature/Unsorted/37157 Door.sql:23]: Missing NpcLooksLikeObject

**Review by ziang4891 — CHANGES_REQUESTED** [review]
Missing NpcLooksLikeObject on all of the objects in 2008-05-Introductions\9 WeenieDefaults\Creature\Unsorted

File: Database/Patches/2008-05-Introductions/9 WeenieDefaults/Creature/Unsorted/37157 Door.sql (hunk 2 above)
Comment (sLackeR-hub) [reply Database/Patches/2008-05-Introductions/9 WeenieDefaults/Creature/Unsorted/37157 Door.sql]: Fixed on all Unsorted.

File: Database/Patches/2008-05-Introductions/8 QuestDefDB/JesterMarkerQuest.sql (hunk 1 above)
Comment (sLackeR-hub) [reply Database/Patches/2008-05-Introductions/8 QuestDefDB/JesterMarkerQuest.sql:2]: Removed from Foolish Ambition.

File: Database/Patches/2008-03-FoolishAmbition/9 WeenieDefaults/Creature/Unsorted/37169 Master's Sentinel.sql [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+     , (37169,   6,         -1) /* ItemsCapacity */
+     , (37169,   7,         -1) /* ContainersCapacity */
+     , (37169,  16,         32) /* ItemUseable - Remote */
+     , (37169,  93,    6292504) /* PhysicsState - ReportCollisions, IgnoreCollisions, Gravity, ReportCollisionsAsEnvironment, EdgeSlide */
+     , (37169,  95,          3) /* RadarBlipColor - White */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (37169,   1, True ) /* Stuck */
+     , (37169,  19, False) /* Attackable */;
+
```
[the remark this answers is not in the document]
Comment (sLackeR-hub) [reply Database/Patches/2008-03-FoolishAmbition/9 WeenieDefaults/Creature/Unsorted/37169 Master's Sentinel.sql:21]: Added.

**Pull request merged**